lib/RateLimiterDynamo.js

```diff
@@ -0,0 +1,422 @@
+const RateLimiterRes = require("./RateLimiterRes");
+const RateLimiterStoreAbstract = require("./RateLimiterStoreAbstract");
+
+class DynamoItem {
+  /**
+   * Create a DynamoItem.
+   * @param {string} rlKey - The key for the rate limiter.
+   * @param {number} points - The number of points.
+   * @param {number} expire - The expiration time in seconds.
+   */
+  constructor(rlKey, points, expire) {
+    this.key = rlKey;
+    this.points = points;
+    this.expire = expire;
+  }
+}
+
+class DynamoTableOpts {
+
+  // Free tier DynamoDB provisioned mode params
+  static DEFAULT_READ_CAPACITY_UNITS = 25;
+  static DEFAULT_WRITE_CAPACITY_UNITS = 25;
+
+  /**
+   * Initializes a new instance of the class.
+   *
+   * @param {number} RCU - the read capacity units
+   * @param {number} WCU - the write capacity units
+   */
+  constructor(RCU, WCU) {
```

Comment:
@Daniel-97 You have never used this constructor in your code. Is it required?

Reply:
Yes you are right, i going to fix this